modules/webgl/src/classes/framebuffer.js

```diff
@@ -433,10 +433,11 @@ export default class Framebuffer extends Resource {
         // behavior we always disable mipmaps.
         mipmaps: false,
         // Set MIN and MAG filtering parameters so mipmaps are not used in sampling.
+        // Use LINEAR so subpixel algos like fxaa work.
         // Set WRAP modes that support NPOT textures too.
         parameters: {
-          [GL.TEXTURE_MIN_FILTER]: GL.NEAREST,
-          [GL.TEXTURE_MAG_FILTER]: GL.NEAREST,
+          [GL.TEXTURE_MIN_FILTER]: GL.LINEAR,
+          [GL.TEXTURE_MAG_FILTER]: GL.LINEAR,
```

Comment:
Can we avoid changing the defaults here? Is this going to break e.g. the picking buffer? Add to documentation?

Reply:
My preference would be to update the defaults to avoid complicating the usage in deck, where you'd have to define the framebuffer attachments explicitly ([like this](https://github.com/uber/luma.gl/blob/master/examples/core/dof/app.js#L300-L316)).
It won't change anything other than post-processing, since it only affects sampling from framebuffer attachments. It won't break picking, because texture filtering doesn't affect `readPixels`.

I'll update the documentation, though, to reflect the new defaults.

Reply:
Also add an entry in upgrade guide.